en/middleware/micrortps_manual_code_generation.md

```diff
@@ -133,3 +133,7 @@ src/modules/micrortps_bridge/micrortps_client
 └── microRTPS_transport.h
  0 directories, 4 files
 ```
+
+## Build and use the code
```

Comment:
1. After a manual generation you can run _make PLATFORM_ for compile as usual and the client is compiled.